importer/src/helpers/utils.js

```diff
@@ -97,9 +98,30 @@ function stripDot(name: string): string {
 export function sortName(name: string): string {
 	const articles = ['a', 'an', 'the', 'los', 'las', 'el', 'la'];
 	const suffixes = [
-		'i', 'ii', 'iii', 'iv', 'v', 'vi', 'vii', 'viii', 'ix', 'x', 'xi',
-		'xii', 'xiii', 'xiv', 'xv', 'jr', 'junior', 'sr', 'senior', 'phd', 'md',
-		'dmd', 'dds', 'esq'
+		'i',
+		'ii',
+		'iii',
+		'iv',
+		'v',
+		'vi',
+		'vii',
+		'viii',
+		'ix',
+		'x',
+		'xi',
+		'xii',
+		'xiii',
+		'xiv',
+		'xv',
+		'jr',
+		'junior',
+		'sr',
+		'senior',
+		'phd',
+		'md',
+		'dmd',
+		'dds',
+		'esq'
```

Comment:
Oof, this is rough.
I'd love to have a code formatter that doesn't impose this sort of thing, it's a bit silly.

Reply:
I'll close this then. Makes no sense to force something we're not comfortable with.